core/controllers/diagnostic_test_player_test.py

```diff
@@ -44,18 +47,20 @@ def setUp(self) -> None:
             [
                 platform_parameter_domain.PlatformParameterRule.from_dict({
                     'filters': [{
-                        'type': 'server_mode',
-                        'conditions': [[
-                            '=',
-                            platform_parameter_domain.FeatureStages.DEV.value]
-                            ]
+                        'type': 'platform_type',
+                        'conditions': [['=', 'Backend']]
                     }],
                     'value_when_matched': True
                 })
             ],
-            False
+            True
```

Comment:
Why change this?

Reply:
This is the default_value, so in case no rules are matched the default values are passed. I would require this as I want the value of the platform param on current server to be True.